src/cli/backup/exchange_e2e_test.go

```diff
@@ -35,6 +35,184 @@ var (
 	events   = path.EventsCategory
 )
 
+// ---------------------------------------------------------------------------
+// tests with azure flags in exchange create
+// ---------------------------------------------------------------------------
+
+type ExchangeCMDWithFlagsE2ESuite struct {
+	tester.Suite
+	acct       account.Account
+	st         storage.Storage
+	vpr        *viper.Viper
+	cfgFP      string
+	repo       repository.Repository
+	m365UserID string
+	recorder   strings.Builder
+}
+
+func TestBackupExchangeInvalidFlagValueSuite(t *testing.T) {
```

Comment:
This should match the name of the suite being initialized:
```suggestion
func TestExchangeCMDWithFlagsE2ESuite(t *testing.T) {
```